mavlink_vehicles/mavlink_vehicles.cc

```diff
@@ -758,6 +758,17 @@ void mav_vehicle::rotate(double angle_deg, bool autocontinue)
     // Update timestamp
```

Comment:
This commit can be squashed in the previous one, just add the explanation that you are changing for brakes and rotation. It's easier to understand the changes if they are together.